posthog/api/__init__.py

```diff
@@ -119,20 +117,11 @@ def api_not_found(request):
 projects_router.register(r"actions", ActionViewSet, "project_actions", ["team_id"])
 projects_router.register(r"groups", ClickhouseGroupsView, "project_groups", ["team_id"])
 projects_router.register(r"groups_types", ClickhouseGroupsTypesView, "project_groups_types", ["team_id"])
-projects_router.register(r"insights", InsightViewSet, "project_insights", ["team_id"])
+projects_router.register(r"insights", ClickhouseInsightsViewSet, "project_insights", ["team_id"])
 projects_router.register(r"cohorts", CohortViewSet, "project_cohorts", ["team_id"])
 projects_router.register(r"persons", PersonViewSet, "project_persons", ["team_id"])
 projects_router.register(r"elements", ElementViewSet, "project_elements", ["team_id"])
 projects_router.register(r"experiments", ClickhouseExperimentsViewSet, "project_experiments", ["team_id"])
 projects_router.register(
     r"session_recordings", ClickhouseSessionRecordingViewSet, "project_session_recordings", ["team_id"],
 )
-
-if settings.EE_AVAILABLE:
-    from ee.clickhouse.views.insights import ClickhouseInsightsViewSet
-
-    projects_router.register(r"insights", ClickhouseInsightsViewSet, "project_insights", ["team_id"])
-else:
-    from posthog.api.insight import InsightViewSet
```

Comment:
Seems like we could get rid of all these PostHog-only classes too, right?

Reply:
You mean Postgres-only? We could! Definitely for another PR though. Also, this perhaps already falls under the umbrella of @EDsCODE's yeeting, but I don't know for sure.